src/borg/helpers.py

```diff
@@ -2277,3 +2277,10 @@ def dash_open(path, mode):
 
 def is_terminal(fd=sys.stdout):
     return hasattr(fd, 'isatty') and fd.isatty() and (sys.platform != 'win32' or 'ANSICON' in os.environ)
+
+
+def umount(mountpoint):
+    try:
+        return subprocess.call(['fusermount', '-u', mountpoint])
```

Comment:
why? -> because I suspect windows will have its own way (as usual) rather than "umount" or "fusermount".

agreed that it's early to think about that considering no native windows code is merged yet.
but considering we already have the platform code, we don't need to remove it if we can already see it might be useful later (and even now, for fusermount linux special case).